source/scss/_pf-basic-icons.scss

```diff
@@ -219,3 +219,23 @@
 .#{$pf-icon-prefix}-zone:before {
   content: $pficon-var-zone;
 }
+
+@mixin pficon-circle($width-height, $font-size, $line-height) {
+  width: $width-height;
+  height: $width-height;
+  font-size: $font-size;
+  line-height: $line-height;
+  text-align: center;
+  border: 2px solid $color-pf-blue-300;
+  border-radius: 50%;
+}
+// line-height value should be 4px less than width-height value to accommodate 2px border on each side
+.pficon-circle-lg {
+  @include pficon-circle(106px, 46px, 102px);
+}
+.pficon-circle-md {
+  @include pficon-circle(54px, 23px, 50px);
+}
```

Comment:
I thought about that. I would need to use specific rem values instead of variables like $font-size-relative-lg. What does everyone think?